Applications/Internet/Internet Explorer 6.0/Online/script.js

```diff
@@ -12,7 +12,7 @@ include(["engines", "wine", "verbs", "msls31"]);
 var installerImplementation = {
     run: function () {
         var appsManager = Bean("repositoryManager");
-        var application = appsManager.getApplication(["Applications", "Internet", "Internet Explorer 6.0"]);
+        var application = appsManager.getApplication([TYPE_ID, CATEGORY_ID, APPLICATION_ID]);
```

Comment:
These lines look like black magic for everyone who doesn't know where the ids come from and what they mean.

Reply:
TYPE_ID refers to either Applications,Engines or Verbs. CATEGORY_ID refers to what inside the aforementioned folders. APPLICATION_ID is self-explainatory. At least thats how I understand it.

Reply:
Yes, that is correct.

The issue with these fields is just that a first time script developer, who looks at this script for reference purposes for example, is likely to stumble over these variables.
The reason being is, that they are set from outside (i.e. the Java part of Phoenicis) and that they  are uncommented (i.e. https://phoenicisorg.github.io/scripts contains no explanation about these constants).

I don't think we can improve the first issue, therefore this comment is meant as a kind of note for the future, meant for us, to think about a more intuitive/understandable solution, how to make these constants accessible.

Reply:
This issue will be solved with https://github.com/PhoenicisOrg/phoenicis/issues/1242.